validator/schema/releases_schema.py

```diff
@@ -81,6 +82,7 @@ def releases_schema(file):
                             Optional('enabled'): bool,
                             Optional('flags'): [str],
                         },
+                        Optional('check_golang_versions'): Or('no', 'exact', 'x.y'),
```

Comment:
Justin made a change to this field to accept boolean value `no/false`: https://github.com/openshift/doozer/commit/58af902ec4df06646554ff7426856591457a9bcc.